web/setup.py

```diff
@@ -496,6 +496,8 @@ def set_prefs(username,
                   sqlite_path: Optional[str] = None,
                   ):
         """Set User preferences."""
+        if not pref_options:
```

Comment:
Can this bug be fixed by setting the optional param pref_options to []?
Like` pref_options: Annotated[Optional[List[str]], typer.Argument()] = [],`

Reply:
Generally function parameters should not have mutable default arguments. Hence I set default value as None.